static/src/js/pages/SearchList/SearchList.jsx

```diff
@@ -174,86 +182,86 @@ const SearchList = ({ limit }) => {
     if (formattedType === conceptTypes.Collections) {
       return [
         {
-          dataKey: 'shortName',
-          title: 'Short Name',
           className: 'col-auto',
           dataAccessorFn: buildEllipsisLinkCell,
-          sortFn
+          dataKey: 'shortName',
+          sortFn,
+          title: 'Short Name'
         },
         {
-          dataKey: 'version',
-          title: 'Version',
+          align: 'end',
           className: 'col-auto text-nowrap',
-          align: 'end'
+          dataKey: 'version',
+          title: 'Version'
         },
         {
-          dataKey: 'title',
-          sortKey: 'entryTitle',
-          title: 'Entry Title',
           className: 'col-auto',
           dataAccessorFn: buildEllipsisTextCell,
-          sortFn: (_, order) => sortFn('entryTitle', order)
+          dataKey: 'title',
+          sortFn: (_, order) => sortFn('entryTitle', order),
+          sortKey: 'entryTitle',
+          title: 'Entry Title'
         },
         {
-          dataKey: 'provider',
-          title: 'Provider',
-          className: 'col-auto text-nowrap',
           align: 'center',
-          sortFn
+          className: 'col-auto text-nowrap',
+          dataKey: 'provider',
+          sortFn,
+          title: 'Provider'
         },
         {
-          dataKey: 'granules.count',
-          title: 'Granule Count',
+          align: 'end',
           className: 'col-auto text-nowrap',
-          align: 'end'
+          dataKey: 'granules.count',
+          title: 'Granule Count'
         },
         {
-          dataKey: 'tagDefinitions',
-          title: 'Tags',
+          align: 'end',
           className: 'col-auto text-nowrap',
           dataAccessorFn: buildTagCell,
-          align: 'end'
+          dataKey: 'tagDefinitions',
+          title: 'Tags'
         },
         {
-          dataKey: 'revisionDate',
-          title: 'Last Modified (UTC)',
+          align: 'end',
           className: 'col-auto text-nowrap',
           dataAccessorFn: (cellData) => moment.utc(cellData).format(DATE_FORMAT),
-          align: 'end',
-          sortFn
+          dataKey: 'revisionDate',
+          sortFn,
+          title: 'Last Modified (UTC)'
         }
       ]
     }
 
     return [
       {
-        dataKey: 'name',
-        title: 'Name',
         className: 'col-auto',
         dataAccessorFn: buildEllipsisLinkCell,
-        sortFn
+        dataKey: 'name',
+        sortFn,
+        title: 'Name'
       },
       {
-        dataKey: 'longName',
-        title: 'Long Name',
         className: 'col-auto',
         dataAccessorFn: buildEllipsisTextCell,
-        sortFn: (_, order) => sortFn('longName', order)
+        dataKey: 'longName',
+        sortFn,
```

Comment:
I did a bunch of sorting but didn't want to let this get lost. Can anyone confirm if this line is needed or not? The test file wanted me to test this line but I couldn't find a use case for it and deleting it did not result in any changes.